package.json

```diff
@@ -41,6 +41,9 @@
       }
     ]
   },
+  "activationEvents": [
+    "onStartupFinished"
```

Comment:
This is needed so that the extension can format right away, before the first run of the command from the Command Palette

Reply:
Thanks, that make sense.